src/widgets/town_widget.h

```diff
@@ -49,6 +49,7 @@ enum TownFoundingWidgets {
 	WID_TF_NEW_TOWN,          ///< Create a new town.
 	WID_TF_RANDOM_TOWN,       ///< Randomly place a town.
 	WID_TF_MANY_RANDOM_TOWNS, ///< Randomly place many towns.
+	WID_TF_EXPAND_ALL_TOWNS,  ///< Make all towns slightly grow.
```

Comment:
Might as well match the tooltip. 🙂 

```suggestion
	WID_TF_EXPAND_ALL_TOWNS,  ///< Make all towns grow slightly.
```